.github/workflows/CI.yml

```diff
@@ -0,0 +1,277 @@
+   REDIS_HOST: localhost
+   REG_VERSION: v2.7.1-patch-2819-2553
+   UI_BUILDER_VERSION: 1.6.0
+
+on: 
+  pull_request:
+  push:
+
+jobs:
+  UTTEST:
+    env:
+       UTTEST: true
+    runs-on: 
+      #- self-hosted
+      - ubuntu-latest
+    steps:
+      - name: Set up Go 1.13
+        uses: actions/setup-go@v1
+        with:
+           go-version: 1.13
+        id: go
+      - name: setup Docker
+        uses: docker-practice/actions-setup-docker@0.0.1
+        with:
+          docker_version: 18.09
+          docker_channel: stable
+      - uses: actions/checkout@v1
+        with:
+         fetch-depth: 1
+         path: src/github.com/goharbor/harbor
+      - name: setup env
+        run: |
+          pwd
+          go env
+          echo "::set-env name=GOPATH::$(go env GOPATH):$RUNNER_WORKSPACE"
+          echo "::add-path::$(go env GOPATH)/bin"
+          echo "::set-env name=TOKEN_PRIVATE_KEY_PATH::${GITHUB_WORKSPACE}/tests/private_key.pem"
+        shell: bash
+      - name: before_install
+        run: |
+          set -x
+          pwd
+          env
+          #sudo apt install -y xvfb
+          #xvfb-run ls
+          curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+          chmod +x docker-compose
+          sudo mv docker-compose /usr/local/bin
+          IP=`hostname -I | awk '{print $1}'`
+          echo '{"insecure-registries" : ["'$IP':5000"]}' | sudo tee /etc/docker/daemon.json
+          echo "::set-env name=IP::$IP"
+          sudo cp ./tests/harbor_ca.crt /usr/local/share/ca-certificates/
+          sudo update-ca-certificates
+          sudo service docker restart
+      - name: install
+        run: |
+          env
+          bash ./tests/showtime.sh ./tests/travis/ut_install.sh
+      - name: script
+        run: |
+          echo IP: $IP
+          bash ./tests/showtime.sh ./tests/travis/ut_run.sh $IP
+        
+  APITEST_DB:
+    env:
+      APITEST_DB: true
+    runs-on:
+      #- self-hosted
+      - ubuntu-latest
+    steps:
+      - name: Set up Go 1.13
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.13
+        id: go
+      - name: setup Docker
+        uses: docker-practice/actions-setup-docker@0.0.1
+        with:
+          docker_version: 18.09
+          docker_channel: stable
+      - uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+          path: src/github.com/goharbor/harbor
+      - name: setup env
+        run: |
+          pwd
+          go env
+          echo "::set-env name=GOPATH::$(go env GOPATH):$RUNNER_WORKSPACE"
+          echo "::add-path::$(go env GOPATH)/bin"
+          echo "::set-env name=TOKEN_PRIVATE_KEY_PATH::${GITHUB_WORKSPACE}/tests/private_key.pem"
+        shell: bash
+      - name: before_install
+        run: |
+          set -x
+          pwd
+          env
+          #sudo apt install -y xvfb
```

Comment:
Is it commented?

Reply:
yes, this is actually not used in my test.
and the installation here for ubuntu will also have problem, so I comment it out